MobileBuy/sample/src/main/java/com/shopify/sample/activity/CollectionListActivity.java

```diff
@@ -91,7 +91,7 @@ protected void onResume() {
 
 
             String initializationMode = getIntent().getStringExtra("optlyInitMode");
-            if (initializationMode.equals("async")) {
+            if (initializationMode == null || initializationMode.equals("async")) {
```

Comment:
@mikeng13 This line was added in your last commit and was preventing the app from running locally. Since async is the default initialization mode in our docs (I think), I made it the default here.